CMakeLists.txt

```diff
@@ -30,74 +33,30 @@ generate_messages(
 catkin_package(
   INCLUDE_DIRS include
   LIBRARIES tue_serialization
-  CATKIN_DEPENDS message_runtime
+  CATKIN_DEPENDS message_runtime std_msgs
 )
 
 ###########
 ## Build ##
 ###########
 
-include_directories(include)
-
-set(HEADER_FILES
-    include/tue/serialization/version.h
-    include/tue/serialization/input_archive.h
-    include/tue/serialization/archive.h
-    include/tue/serialization/conversions.h
-    include/tue/serialization/output_archive.h
-    include/tue/serialization/filesystem.h
+include_directories(
+  include
+  ${catkin_INCLUDE_DIRS}
 )
 
+file(GLOB_RECURSE HEADER_FILES include/*.h)
```

Comment:
Can be removed. Seems like it is not being used anywhere

Reply:
By defining this, QtCreator will show these files.

Reply:
Okay.